src/viewProvider.ts

```diff
@@ -174,6 +174,43 @@ export class OceanProtocolViewProvider implements vscode.WebviewViewProvider {
                   }
               });
           </script>
+          <h3>Download Asset</h3>
+
+          <label for="assetDidInput">Asset DID</label>
+          <input id="assetDidInput" placeholder="Enter your asset DID" />
+
+          <label for="filePathInput">File Path</label>
```

Comment:
by the way, `file path` is a bit of a confusing name as they don't have the file yet. It might be better to say `download path` to indicate this is where the file is going to be saved.